modules/generator/registry/native_histogram.go

```diff
@@ -90,6 +104,12 @@ func newNativeHistogram(name string, buckets []float64, onAddSeries func(uint32)
 		traceIDLabelName:  traceIDLabelName,
 		buckets:           buckets,
 		histogramOverride: histogramOverride,
+		externalLabels:    externalLabels,
+
+		// classic
+		nameCount:  fmt.Sprintf("%s_count", name),
```

Comment:
does `name` need to be sanitized for prom? does it ever come from an external config?

Reply:
I think we're good on the name.  Each metric name is a constant.
https://github.com/zalegrala/tempo/blob/f367ceadb5d9c679ae13cadef8085833fef5123a/modules/generator/processor/spanmetrics/spanmetrics.go#L85
https://github.com/zalegrala/tempo/blob/f367ceadb5d9c679ae13cadef8085833fef5123a/modules/generator/processor/servicegraphs/servicegraphs.go#L123
Good question though.